src/main/kotlin/no/nav/helse/flex/controller/SoknadController.kt

```diff
@@ -95,7 +95,7 @@ class SoknadController(
         soknadFraBase.validerSvarPaSoknad()
         val eldsteSoknaden = hentSoknadService.hentEldsteSoknaden(identer, soknadFraBase.fom)
         if (eldsteSoknaden != null && eldsteSoknaden != soknadFraBase.id) {
-            log.warn("Vi fant en eldre søknad med id: $eldsteSoknaden")
+            throw FeilStatusForOppdaterSporsmalException("Vi fant en eldre søknad med id: $eldsteSoknaden. Dette skal ikke skje!")
```

Comment:
Kanskje lag en egen exception for dette caset? Og så tenker jeg at vi kan ta med id på den aktuelle søknaden som forsøkes å sendes .

Det er vel også mulig å lage en test som sjekker at backend gir http 400 når dette slår til.

Reply:
🛠️